src/db/sqlite.rs

```diff
@@ -217,10 +218,12 @@ impl super::Database for Database {
                         "})
                     .help("try again with the latest version of Broker")
             }
-            Some(db_version) if current_version > db_version => self
-                .update_db_version(&current_version)
-                .await
-                .change_context(super::Error::Interact),
+            Some(db_version) if current_version > db_version => {
+                warn!("Broker currently runnning on {db_version:?}. Updating Broker to run on {current_version:?}");
```

Comment:
This log statement is not representative of what is happening here, and isn't really a warning (no user cares about this, it's not an error, and if they did there's nothing they could do about it).